evap/static/ts/src/contact_modal.ts

```diff
@@ -0,0 +1,64 @@
+declare const csrf: typeof import("./csrf-utils");
+declare const bootstrap: typeof import("bootstrap");
+
+class ContactModalLogic {
+    private readonly modal: bootstrap.Modal;
+    private readonly successMessageModal: bootstrap.Modal;
+    private readonly actionButtonElement: HTMLButtonElement;
+    private readonly messageTextElement: HTMLInputElement;
+    private readonly modal_id: string;
+    private readonly title: string;
+
+    constructor(modal_id: string, title: string) {
+        this.modal_id = modal_id;
+        this.title = title;
+        this.modal = new bootstrap.Modal(<HTMLElement>document.getElementById(modal_id));
+        this.successMessageModal = new bootstrap.Modal(
+            <HTMLElement>document.getElementById("successMessageModal_" + modal_id),
```

Comment:
```suggestion
            document.getElementById<HTMLElement>("successMessageModal_" + modal_id),
```

Reply:
Does not work. In this case, we can use the ! syntax